opencodeblocks/scene/ipynb_conversion.py

```diff
@@ -0,0 +1,241 @@
+""" Module for converting ipynb data to ipyg data """
+
+from typing import Generator, OrderedDict, List, Dict
+
+MARGIN_X: float = 50
+MARGIN_BETWEEN_BLOCKS_X: float = 50
+MARGIN_Y: float = 50
+MARGIN_BETWEEN_BLOCKS_Y: float = 5
+BLOCK_MIN_WIDTH: float = 400
+TITLE_MAX_LENGTH: int = 60
+SOCKET_HEIGHT: float = 44.0
+TEXT_SIZE: float = 12
+TEXT_SIZE_TO_WIDTH_RATIO: float = 0.7
+TEXT_SIZE_TO_HEIGHT_RATIO: float = 1.42
+
+BLOCK_TYPE_TO_NAME: Dict[str, str] = {
+    "code": "OCBCodeBlock",
+    "markdown": "OCBMarkdownBlock",
+}
+
+
+def ipynb_to_ipyg(data: OrderedDict) -> OrderedDict:
+    """Convert ipynb data (ipynb file, as ordered dict) into ipyg data (ipyg, as ordered dict)"""
+
+    blocks: List[OrderedDict] = get_blocks(data)
```

Comment:
```suggestion
    blocks_data: List[OrderedDict] = get_blocks_data(data)
```

blocks_data should be used instead of blocks. blocks should referer to actual OCBBlock instances